cmd/admin-replicate-remove.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/colorjson"
+	"github.com/minio/madmin-go"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/pkg/console"
+)
+
+var adminReplicateRemoveFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "all",
+		Usage: "remove site replication from all participating sites",
+	},
+	cli.BoolFlag{
+		Name:  "force",
+		Usage: "force removal of site(s) from site replication configuration",
+	},
+}
+
+var adminReplicateRemoveCmd = cli.Command{
+	Name:         "remove",
+	Usage:        "remove one or more sites from site replication",
+	Action:       mainAdminReplicationRemoveStatus,
+	OnUsageError: onUsageError,
+	Before:       setGlobalsFromContext,
+	Flags:        append(globalFlags, adminReplicateRemoveFlags...),
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+
+EXAMPLES:
+  1. Remove site replication for all sites:
+     {{.Prompt}} {{.HelpName}} minio2 --all --force
+
+  2. Remove site replication for site with site names alpha, baker from active cluster minio2:
+     {{.Prompt}} {{.HelpName}} minio2 alpha baker
+`,
+}
+
+type srRemoveStatus madmin.ReplicateRemoveStatus
+
+func (i srRemoveStatus) JSON() string {
+	ds, e := json.MarshalIndent(madmin.ReplicateRemoveStatus(i), "", " ")
+	fatalIf(probe.NewError(e), "Unable to marshal into JSON.")
+	return string(ds)
+}
+
+func (i srRemoveStatus) String() string {
+	return console.Colorize("UserMessage", i.Status)
```

Comment:
```suggestion
	return console.Colorize("UserMessage", i.Status)
```

We should word it like what all sites were removed?

Reply:
the Status here is sent from the server and will have more detailed info on which sites were not removed, if any

Reply:
Is it not possible to show what was asked to be removed since you get the input for the site name?